src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/ConsoleLoggerExtensionsTests.cs

```diff
@@ -597,6 +597,7 @@ private static void VerifyHasOnlySimpleProperties(Type type)
                 // or else NativeAOT would break
                 Assert.True(prop.PropertyType == typeof(string) ||
                     prop.PropertyType == typeof(bool) ||
+					prop.PropertyType == typeof(char) ||
```

Comment:
Why was this change necessary?

Reply:
Idem, we have added the `char` type for the new `IndentCharacter` property of the `JsonWriterOptions` class.